client/my-sites/domains/domain-management/list/style.scss

```diff
@@ -7,6 +7,35 @@
 	margin-bottom: 2px;
 }
 
+.list__header-primary-domain {
+	display: flex;
+
+	.list__header-primary-domain-info {
+		text-align: left;
```

Comment:
The "Primary domain" header should be 14px (or better use the following snippet):
```font-size: $font-body-small;```